RoboFile.php

```diff
@@ -19,15 +19,17 @@ public function checkCodestyle($sniffersPath = null)
 	{
 		if (is_null($sniffersPath))
 		{
-			$sniffersPath = __DIR__ . '/.travis/phpcs/Joomla';
+			$sniffersPath = __DIR__ . '/.travis/phpcs/';
 		}
 
 		$this->taskCodeChecks()
 			->setBaseRepositoryPath(__DIR__)
+			->setCodeStyleStandardsRepo('photodude/coding-standards')
```

Comment:
You can use the provided `vendor/greencape/coding-standards/src` folder to fix this (no need to specify a branch)